config/dependencies/istio/sail/istio.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: operator.istio.io/v1alpha1
+kind: Istio
+metadata:
+  name: istiocontrolplane
+  namespace: istio-system
+spec:
+  version: v1.20.0
+  values:
+    pilot:
+      autoscaleEnabled: false
+  rawValues:
+    gateways:
+      istio-ingressgateway:
+        autoscaleEnabled: false
```

Comment:
be good to get a couple of comments as to what it does or why its needed?

Reply:
This was just to match our original local deploy configuration but isn't necessary and could be removed https://github.com/Kuadrant/kuadrant-operator/blob/main/config/dependencies/istio/istio-operator.yaml#L48-L54